### PR TITLE
[TRAFODION-2665] Update Character String Data Types

### PR DESCRIPTION
**Pull request by liuyu000**
Changes proposed in this pull request:

1. Add descriptions for `unit` and `char-set-name`.
2. Fix some mistakes based on AsciiDoc syntax.

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc [hunk 1]
```diff
@@ -649,18 +664,18 @@ shorter than the maximum length, but the maximum internal size of a
...
-* `VARCHAR (_length_) [_char-set_] [UPSHIFT] [[NOT]CASESPECIFIC]`
+* `VARCHAR (_length_ [_unit_]) [_char-set_] [UPSHIFT] [[NOT]CASESPECIFIC]`
 +
 specifies a column with varying-length character data. VARCHAR is
 equivalent to data type CHAR[ACTER] VARYING.
 
 * `NCHAR [(_length_)] [UPSHIFT] [[NOT]CASESPECIFIC], NATIONAL CHAR[ACTER] [(_length_)] [UPSHIFT] [[NOT]CASESPECIFIC]`
 +
-specifies a column with data in the predefined national character set.
+specifies a column with data in the predefined national character set(USC2).
```
Comment (zellerh) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: This should say (UCS2). Maybe also add a space between "set" and (UCS2).

File: docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc [hunk 2]
```diff
@@ -649,18 +664,18 @@ shorter than the maximum length, but the maximum internal size of a
...
 
 * `NCHAR VARYING [(_length_)] [UPSHIFT] [[NOT]CASESPECIFIC], NATIONAL CHAR[ACTER] VARYING (_length_) [UPSHIFT] [[NOT]CASESPECIFIC]`
 +
-specifies a column with varying-length data in the predefined national character set.
+specifies a column with varying-length data in the predefined national character set(USC2).
```
Comment (zellerh) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: Same here.

File: docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc [hunk 3]
```diff
@@ -604,39 +604,54 @@ numeric, datetime, or interval data.
...
 +
-is a positive integer that specifies the number of characters allowed in
+is a positive integer that specifies the number of characters (or bytes, see below) allowed in
 the column. You must specify a value for _length_.
 
-* `_char-set-name_`
+* `_unit_`
++
+is an optional unit of either CHAR[ACTER[S]] or BYTE[S]. This unit is meaningful only for UTF8 characters.
+A UTF8 character is one to four bytes in length, therefore the storage length of a CHAR column that can hold _n_ UTF8 characters is 4*_n_ bytes.
```
Comment (zellerh) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: Sorry, my mistake, this does not mention what the default unit is. Could we add something like: The default unit is CHARACTERS.

Comment (liuyu000) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: Thanks Hans. I've incorporated your comment. :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc (hunk 1 above)
Comment (liuyu000) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: Thanks Hans. I've incorporated your comment. :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc (hunk 2 above)
Comment (liuyu000) [inline docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc]: Thanks Hans. I've incorporated your comment. :)

**Comment by zellerh** [comment]
+1

**Pull request merged**